pkg/antctl/raw/multicluster/resourceimport.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	k8sscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	mcsscheme "sigs.k8s.io/mcs-api/pkg/client/clientset/versioned/scheme"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	antreamcscheme "antrea.io/antrea/multicluster/pkg/client/clientset/versioned/scheme"
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+var command *cobra.Command
+
+type resourceImportOptions struct {
+	namespace    string
+	outputFormat string
+	allNamespace bool
+}
+
+var options *resourceImportOptions
+
+var resourceImportExample = strings.Trim(`
+Gel all resource imports of cluster set in default namesapce
+$ antctl get resourceimport
+Get all resource imports of cluster set in all namespace
+$ antctl get resoruceimport -A
+Get all resource imports of specified namespace
+$ antctl get resourceimport -n <NAMESPACE>
+Get all resource imports and print json format
+$ antctl get resourceimport -o json
+`, "\n")
+
+func (o *resourceImportOptions) validateAndComplete() error {
+	if o.allNamespace {
+		o.namespace = metav1.NamespaceAll
+	} else if o.namespace == "" {
+		o.namespace = metav1.NamespaceDefault
+	}
+
+	return nil
+}
+
+func init() {
+	command = &cobra.Command{
+		Use: "resourceimport",
+		Aliases: []string{
+			"resourceimports",
+			"ri",
+		},
+		Short:   "Print Multi-Cluster ResourceImports",
+		Args:    cobra.MaximumNArgs(1),
```

Comment:
I don't see any reference to this positional argument in help messages

Reply:
done